src/gui/guiiconprovider.cpp

```diff
@@ -117,12 +129,13 @@ QIcon GuiIconProvider::generateDifferentSizes(const QIcon &icon)
 
     return newIcon;
 }
+
 #endif
 
-QString GuiIconProvider::getIconPath(const QString &iconId)
+QString GuiIconProvider::getIconPath(const QString &iconId, const QString &overrideTheme)
```

Comment:
Your changes seem to need a rebasing on top of the current master (the signature of this function is old).